### ISSUE TITLE
Adding support for PlatformIO ESP8266 board using the Arduino framework

### ISSUE THREAD

**Issue by omar-bb** [danielaparker/jsoncons#274]
Dear Daniel,

**Can we have PlatformIO ESP8266 ESP-01 512K board using the Arduino framework support**

**The actual library doesn't support this kind of compiler it is really sad because this lib can be really useful for esp users**

Thanks for doing such an amazing work!

**Comment by danielaparker** [comment]
I don't know how to go about that, as I don't have an Arduino environment for testing.

**Comment by omar-bb** [comment]
Hey sorry for the late response, I think that there is some Arduino emulator out there, but you can use platformio to compile your code without uploading it to an actual board.

**Comment by danielaparker** [comment]
Okay, I've installed platformio, and will have a look. Could take a while!

**Comment by omar-bb** [comment]
Thank you very much for taking the time to answer me, I wish you good luck and I will help you if necessary!

**Comment by danielaparker** [comment]
Let's keep this one open.

**Comment by caphadoc** [comment]
Hello Daniel,

have you had time to look at this request? I would really like to have your library on the ESP8266 for my connected greenhouse project :)

Thx

**Comment by danielaparker** [comment]
No, sorry. Right now, the only thing I could offer is that if you make the attempt and identify specific compilation errors with the existing code,  I could look at workarounds.
